inc/installers/class-migrator.php

```
@@ -459,7 +459,7 @@
 	 * @param string        $installer The installer name.
 	 * @param object        $wizard Wizard class.
 	 *
-	 * @return bool|WP_Error
+	 * @return void
 	 */
 	public function handle($status, $installer, $wizard) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$status'. (undefined)

(UnusedFormalParameter)